File: auto-docs/executables/r/horizontal_histogram.r
# Learn about API authentication here: {BASE_URL}/r/getting-started
# Find your api_key here: {BASE_URL}/settings/api

library(plotly)

y = rnorm(500)

py <- plotly(username='TestBot', key='r1neazxo9w')
data <- list(
  list(
    y = y, 
    type = "histogram"
  )
)
response <- py$plotly(data, kwargs=list(filename="horizontal-histogram", fileopt="overwrite"))
url <- response$url

File: auto-docs/executables/julia/horizontal_histogram.jl
# Learn about API authentication here: {BASE_URL}/julia/getting-started
# Find your api_key here: {BASE_URL}/settings/api

using Plotly

y = randn(500)

Plotly.signin("TestBot", "r1neazxo9w")
data = [
  [
    "y" => y, 
    "type" => "histogram"
  ]
]
response = Plotly.plot(data, ["filename" => "horizontal-histogram", "fileopt" => "overwrite"])
plot_url = response["url"]
